DockerOrchestration/TendermintCore/config.toml

```diff
@@ -0,0 +1,11 @@
+# This is a TOML config file.
```

Comment:
no need for config either if it replicates default config. TM will create one if it is missing